util/src/commonTest/kotlin/com/wire/kalium/util/DateTimeUtilBenchmark.kt

```diff
@@ -0,0 +1,30 @@
+package com.wire.kalium.util
+
+import com.wire.kalium.util.DateTimeUtil.toIsoDateTimeString
+import kotlinx.coroutines.test.runTest
+import kotlinx.datetime.Instant
+import kotlin.random.Random
+import kotlin.test.Ignore
+import kotlin.test.Test
+import kotlin.time.ExperimentalTime
+import kotlin.time.measureTime
+
+@Ignore
```

Comment:
maybe we can delete the DateTimeUtilBenchmark since it is Ignored and have no assertion ?

Reply:
Just like we have the benchmarks for database, we could have for this as well.

It's normal to ignore while we have a proper setup, otherwise we're wasting CI time